.github/workflows/azure-ci.yml

```diff
@@ -62,13 +64,16 @@ jobs:
         run: |
           ./cloudbeat -c deploy/azure/cloudbeat-azure.yml -d '*' &
 
+      - name: Wait for cloudbeat to send events
+        run: sleep 264
```

Comment:
This jumps out at me as a very specific numer. Out of curiosity, is that an off-the-wall value, or does it correspond to some specific timeout?

Reply:
Just off-the-wall.
I wanted to give some time for `cloudbeat` to send findings.
We can change it, I will add more tests and see what is the minimal time needed to get findings.

Reply:
Thank you for clarifying.